src/OpenTK.GLControl/GLControl.cs

```diff
@@ -140,6 +150,11 @@ private void ValidateContext(string message)
 
         private void ValidateState()
         {
+            if (FailedCreateContext)
```

Comment:
We want to avoid this kind of branching if possible. A "null object" should work. See the later comment.

Reply:
Will check see reply below